code/workspaces/web-app/src/components/stacks/StackCards.js

```diff
@@ -103,4 +105,5 @@ StackCards.propTypes = {
   editPermission: PropTypes.string,
   scalePermission: PropTypes.string,
   copySnippets: PropTypes.objectOf(PropTypes.func),
+  showUsernames: PropTypes.bool.isRequired,
```

Comment:
There are several places in `containers` (such as under `adminListUsers`, `ProjectsContainer`, `DataStorageContainer` and `ProjectClustersContainer`) where `StackCards` is called directly rather than through `StacksContainer` - do those call sites need to be updated as well? Presumably they'll get `showUsername` set as `undefined` and therefore hide the usernames with no option to show them.

Reply:
Good point, this should not be required - as long as the state is set to false by default it doesn't need to be specified everywhere.